Multiple Lambda Functions Causes Build Failure

Steps to reproduce:

1. Create a new project from lambda-builder: 
```
curl -O https://raw.githubusercontent.com/knockrentals/lambda-builder/master/Makefile && make project
```
2. Add another function: 
```
mkdir functions/example && touch functions/example/test.py
```
3. `make build`

Expected outcome is to produce two `.zip` files; one for `AppHelloWorld.zip` and `AppExampleTest.zip` but instead I get the following error message:

```
make: *** No rule to make target 'build/functions/AppExampleTestAppHelloWorld.zip', needed by 'build-packages'.  Stop.
```

--- a/Makefile
+++ b/Makefile
@@ -122,7 +122,7 @@ upcase = $(shell echo "$1" | tr '[:lower:]' '[:upper:]')
 # Upcases first letter of each word
 titleize = $(shell STR="$(call downcase,$1)"; LIST=( $$STR ); echo "$${LIST[@]^}")
 # Converts $1: pathlist into a camelcased words
-camelize-path = $(subst $(space),,$(call titleize,$(subst /,$(space),$1)))
+camelize-path = $(foreach path,$(call normalize,$1),$(subst $(space),,$(call titleize,$(subst /,$(space),${path}))))
 # Converts $1: pathlist into proper lambda function names
 fun-name-from-dir = $(call camelize-path,$(call dir-merge,${APP},$1))
 

